app.js

```diff
@@ -1,8 +1,23 @@
 import { getDogs } from './fetch-utils.js';
 import { renderDogCard } from './render-utils.js';
+// import { dogSearch } from './fetch-utils.js';
 
+// const form = document.querySelector('form');
 const dogListContainer = document.getElementById('dog-list-container');
 
 // on load
-// fetch all dogs
-// render and append all dog cards to the container
+
+
+window.addEventListener('load', async() => {
+    const dogs = await getDogs();
+
+    for (let dog of dogs) {
+        const dogEl = renderDogCard(dog);
+
+        dogListContainer.append(dogEl);
+
+        console.log(dogs);
```

Comment:
remember to clean up console.logs like these for submission. The one below in the fetch-utils is fine though since thats doing our error handling for the API call.